build.gradle

```diff
@@ -607,6 +607,8 @@ project(':iceberg-hive-runtime') {
     if (jdkVersion == '8') {
       compile project(':iceberg-hive3')
     }
+    // including thrift so that the Tez AM can communicate with the HMS when using Hive catalog
+    compile("org.apache.thrift:libfb303:0.9.3")
```

Comment:
Shouldn't we use the versions.props to define the required version?

Reply:
Could do. It's not currently defined there, and both `mr` (hive2) and `hive3` use the 0.9.3 version. But to enforce consistency, I'll move it to version.props.